modules/lldp/control.c

```diff
@@ -0,0 +1,274 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+
+#include "gr_lldp.h"
+#include "lldp_priv.h"
+
+#include <gr_api.h>
+#include <gr_control.h>
+#include <gr_graph.h>
+#include <gr_iface.h>
+#include <gr_log.h>
+#include <gr_port.h>
+#include <gr_worker.h>
+
+#include <event2/event.h>
+#include <rte_common.h>
+#include <rte_ethdev.h>
+#include <rte_graph_worker.h>
+#include <rte_version.h>
+
+static struct rte_ether_addr lldp_dst0 = {
+	.addr_bytes = {0x01, 0x80, 0xc2, 0x00, 0x00, 0x00},
+};
+static struct rte_ether_addr lldp_dst1 = {
+	.addr_bytes = {0x01, 0x80, 0xc2, 0x00, 0x00, 0x03},
+};
+static struct rte_ether_addr lldp_dst2 = {
+	.addr_bytes = {0x01, 0x80, 0xc2, 0x00, 0x00, 0x0e},
+};
+
```

Comment:
If these are only defined to be called in `lldp_iface_event`, it would be better to have them as stack variables.